### DISCUSSION TITLE
Is there any point in installing uwebsockets.js on bun.sh?

### DISCUSSION THREAD

**Discussion by johnnyApplePRNG** [uNetworking/uWebSockets.js#1021] [Q&A] [answered]
According to their little AI help chatbot... There is no point to importing uwebsockets.js... but our main github page suggests to do just that.
![Screenshot_20240215_003023_Firefox](https://github.com/uNetworking/uWebSockets.js/assets/86027304/f9295e38-cb6f-49ee-9f87-46b732ce4f9a)

**Comment by uNetworkingAB** [comment, marked as answer]
Ah,

> bun install uNetworking/uWebSockets.js#v20.41.0

bun install is like npm install but less shitty. It's not installing for bun, it's installing using bun. Like npm but not pure diarrhea
